ledger/ledger-api-integration-tests/src/test/itsuite/scala/com/digitalasset/platform/tests/integration/ledger/api/commands/CommandCompletionServiceIT.scala

```diff
@@ -191,7 +191,10 @@ class CommandCompletionServiceIT
         def tailCompletions(
             context: LedgerContext
         ): Future[Completion] = {
-          val (streamObserver, future) = FirstElementObserver[CompletionStreamResponse]
+          val (streamObserver, future) = FirstElementObserver.filter[CompletionStreamResponse] {
```

Comment:
why not use `collect`?

Reply:
nevermind, I thought it's a reactive stream.

Reply:
You mean as a name?